Examples/SwiftExample/OneSignalDemo/AppDelegate.swift

```diff
@@ -47,6 +47,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, OSPermissionObserver, OSS
             print("Received Notification: ", notification.notificationId ?? "no id")
             print("launchURL: ", notification.launchURL ?? "no launch url")
             print("content_available = \(notification.contentAvailable)")
+            if notification.notificationId == "example_silent_notif" {
```

Comment:
A better example might be checking for something in `notification.additionalData` instead of checking `notification.notificationId`